docs/src/main/sphinx/connector/clickhouse.md

```diff
@@ -41,7 +41,11 @@ connection-password=examplepassword
 
 The `connection-url` defines the connection information and parameters to pass
 to the ClickHouse JDBC driver. The supported parameters for the URL are
-available in the [ClickHouse JDBC driver configuration](https://clickhouse.com/docs/en/integrations/java#configuration).
+available in the [ClickHouse JDBC driver
+configuration](https://clickhouse.com/docs/en/integrations/java#configuration).
+When using ClickHouse 24.3 or higher, ensure to set `externalDatabase=false` as
+part of the JDBC connection parameters to avoid performance issues for fetching
+schema information.
```

Comment:
`externalDatabase=false` is set automatically:
https://github.com/trinodb/trino/pull/24515/commits/e605a050c63bf4eeb560e94d18a46083ef3c576f#diff-41ee697e2cf6355e8a9bf591e67564a67146f54ca478348a347b8ba7a7663083R68-R72
Also we do not support older unsupported versions any more.

Reply:
Cool.. then we dont need this PR .. I am closing it.